### PR TITLE
Fixing build

### PR DESCRIPTION
**Pull request by CarlosGamero**
## Changes

Please describe

## Checklist

- [x] Apply one of following labels; `major`, `minor`, `patch` or `skip-release`
- [x] I've updated the documentation, or no changes were necessary
- [x] I've updated the tests, or no changes were necessary

### CODE REVIEW

File: packages/app/supported-languages/package.json [hunk 1]
```diff
@@ -1,52 +1,49 @@
...
+    "lint": "eslint --cache --max-warnings=0 . && prettier --check --log-level warn src \"**/*.{json,md,ts,tsx}\" && tsc --noEmit",
+    "lint:fix": "prettier --write src \"**/*.{json,md,ts,tsx}\" --log-level=warn && eslint . --fix",
+    "test:ci": "vitest run --coverage",
+    "prepublishOnly": "npm run build",
+    "package-version": "echo $npm_package_version"
+  },
+  "devDependencies": {
+    "@lokalise/eslint-config": "latest",
+    "@lokalise/prettier-config": "latest",
+    "@lokalise/node-core": "^11.2.0",
```
Comment (CarlosGamero) [inline packages/app/supported-languages/package.json:39]: node-core was missing and we are using a type from there

File: packages/app/api-common/package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
 		"prettier": "3.3.3",
 		"rimraf": "^5.0.5",
 		"typescript": "5.5.3",
-		"vite": "^5.3.3",
+		"vite": "5.3.4",
```
Comment (CarlosGamero) [inline packages/app/api-common/package.json:46]: This causes to have in node-modules a non-compatible version with the one on out vite package

File: packages/dev/package-vite-config/package.json [hunk 3]
```diff
@@ -31,8 +31,7 @@
   },
   "dependencies": {
     "vite": "5.3.4",
-    "vite-plugin-dts": "3.9.1",
-    "vitest": "^2.0.3"
```
Comment (CarlosGamero) [inline packages/dev/package-vite-config/package.json:35]: I think vitest is not needed here

Comment (kibertoad) [reply packages/dev/package-vite-config/package.json:35]: it's referenced in the config itself, see:
```ts
import { type UserConfig, defineConfig } from 'vitest/config'
```

**Review by kibertoad — APPROVED** [review]

File: packages/dev/package-vite-config/package.json (hunk 3 above)
Comment (CarlosGamero) [reply packages/dev/package-vite-config/package.json]: Good catch! sorry I missed it 😓 added it back

**Review by kibertoad — APPROVED** [review]

**Pull request merged**